src/main/java/tasks/Todo.java

```diff
@@ -0,0 +1,29 @@
+package tasks;
+
+public class Todo extends Task {
+    private static final String CODE = "T";
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    public Todo(boolean done, String description) {
+        super(done, description);
+    }
+
+    @Override
+    public String getCode() {
+        return CODE;
+    }
+
+    @Override
+    public String getDatetime() {
+        return "";
+    }
```

Comment:
Looks a bit strange to give an empty string for the date time.
Perhaps there may be some issues with the hierarchy.

Reply:
Noted, i will bring the implementation of getDatetime() to Task class perhaps.